docs/source/install/install.rst

```diff
@@ -69,15 +69,22 @@ Please feel free to `report <https://github.com/openPMD/openPMD-api/issues/new/c
 
 .. code-block:: bash
 
-   # optional:             --user
-   pip install openpmd-api
+   # we need pip 19 or newer
+   # optional:                   --user
+   python3 -m pip install -U pip
+
+   # optional:                        --user
+   python3 -m pip install openpmd-api
 
 or with MPI support:
 
 .. code-block:: bash
 
-   # optional:                                                  --user
-   openPMD_USE_MPI=ON pip install openpmd-api --no-binary :all:
+   # optional:                   --user
+   python3 -m pip install -U pip
+
+   # optional:                                                                   --user
+   openPMD_USE_MPI=ON python3 -m pip install openpmd-api --no-binary openpmd-api
```

Comment:
I am afraid that, so far this one is not exactly 100% operational with the MPI support in my (frustrating) computer ...

Reply:
Let's take a look! :)

Reply:
Ok, we found a work-around for your problem which is unrelated to the instructions herein for now: https://github.com/openPMD/openPMD-api/issues/593#issuecomment-552690470